test-collateral/populate-test-database.sh

```diff
@@ -7,4 +7,5 @@ echo "INSERT INTO firmware_versions VALUES(2, 'psa', '0.3.0', 'deadbeefdeadbeefd
 pcr0=`cat ../nitro-root-enclave/PCR0`
 echo "INSERT INTO firmware_versions VALUES(3, 'nitro', '0.1.0', '${pcr0}');" >> tmp.sql
 sqlite3 proxy-attestation-server.db < tmp.sql
```

Comment:
Reading this script, one might assume it is run in the `test-collateral/` directory. Running it from `veracruz-test/` and `veracruz-server-test/` is a straightforward step from how it was before, and certainly works, but an alternative approach would be for this script to take the output path as an argument, and be invoked by the Makefile like

```sh
cd veracruz-test && \
		bash ../test-collateral/populate-test-database.sh $@
```

Reply:
That would be an alternative approach. As it works as-is, I'm going to keep it this way for now, unless you see a problem with the way it is.

Reply:
I've changed populate-test-database.sh in https://github.com/veracruz-project/veracruz/pull/75 to accept a path and run-from-anywhere.

Also Diesel seems to have issues running outside of a rust crate, so added a cd into proxy-attestation-server. Not the most robust, but works in our Docker image.